tethys_portal/urls.py

```diff
@@ -38,13 +40,23 @@
 # ensure at least staff users logged in before accessing admin login page
 from django.contrib.admin.views.decorators import staff_member_required
 
+prefix_path = ""
+if hasattr(settings, "PREFIX_TO_PATH") and settings.PREFIX_TO_PATH:
+    prefix_path = f"{settings.PREFIX_TO_PATH}"
+
+
 admin.site.login = staff_member_required(
-    admin.site.login, redirect_field_name="", login_url="/accounts/login/"
+    admin.site.login,
+    redirect_field_name="",
+    login_url=f"/{prefix_path}/accounts/login/",
```

Comment:
When `prefix_path` is `""`, does this result in a url like: `"//accounts/login/"`? Does that cause issues? Also, what about `/`s in the `prefix_path` itself?

Reply:
I have used the following to avoid such conflicts:
```python
LOGIN_URL = portal_config_settings.pop("LOGIN_URL", "/accounts/login/")
PREFIX_URL = portal_config_settings.pop("PREFIX_URL", "/")
if PREFIX_URL is not None and PREFIX_URL != "/":
    PREFIX_URL = f"{PREFIX_URL.strip('/')}"
    STATIC_URL = f"/{PREFIX_URL}/{STATIC_URL.strip('/')}/"
    LOGIN_URL = f"/{PREFIX_URL}/{LOGIN_URL.strip('/')}/"
    FIDO_LOGIN_URL = f"/{PREFIX_URL}/{FIDO_LOGIN_URL.strip('/')}/"
````